api/src/main/java/com/capitalone/dashboard/rest/FeatureController.java

```diff
@@ -1,21 +1,22 @@
 package com.capitalone.dashboard.rest;
 
-import com.capitalone.dashboard.model.DataResponse;
-import com.capitalone.dashboard.model.Feature;
-import com.capitalone.dashboard.model.SprintEstimate;
-import com.capitalone.dashboard.service.FeatureService;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+
+import java.util.List;
+import java.util.Optional;
+
 import org.bson.types.ObjectId;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
-import java.util.List;
-import java.util.Optional;
-
-import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
-import static org.springframework.web.bind.annotation.RequestMethod.GET;
+import com.capitalone.dashboard.model.DataResponse;
+import com.capitalone.dashboard.model.Feature;
+import com.capitalone.dashboard.model.SprintEstimate;
+import com.capitalone.dashboard.service.FeatureService;
 
 /**
  * REST service managing all requests to the feature repository.
```

Comment:
@rob-miller-777 If no changes were made to this class, please take it out of the PR.